common/djangoapps/contentserver/test/test_contentserver.py

```diff
@@ -32,6 +33,10 @@
 TEST_DATA_DIR = settings.COMMON_TEST_DATA_ROOT
 
 
+def get_versioned_asset_url(url):
+    return '/assets/' + md5.new(url).hexdigest() + '/' + url
```

Comment:
This isn't more-unique than the URL itself, is it? The md5 hash is just running on the url, so when the url changes, the hash changes (but the content isn't involved).